packages/transformer/src/IModelTransformer.ts

```diff
@@ -2421,14 +2421,16 @@ export class IModelTransformer extends IModelExportHandler {
         targetId: deletedRelData.targetIdInTarget,
       } as SourceAndTarget);
 
-    // FIXME: make importer.deleteRelationship not need full props
     const targetRelationship = this.targetDb.relationships.tryGetInstance(
       deletedRelData.classFullName,
       relArg
     );
```

Comment:
can't we eliminate this query now and just pass the same args directly to `deleteRelationship`?

Reply:
yup good call, I changed it in both places we call importer.deleteRelationship